.github/workflows/main.yml

```diff
@@ -113,7 +113,7 @@ jobs:
       uses: actions/checkout@v4
 
     - name: Build and run dev container task
-      uses: devcontainers/ci@v0.4
+      uses: devcontainers/ci@v0.3
```

Comment:
Maybe use [v0.3.1900000349](https://github.com/devcontainers/ci/releases/tag/v0.3.1900000349)?

Reply:
I saw, that, but what should be the nomenclature there? all these numbers?

Reply:
https://github.com/devcontainers/ci/tree/chrmarti/federated-ids?tab=readme-ov-file#changelog

Reply:
Also I have an image already. https://github.com/cschuchardt88/neo/pkgs/container/neo-cli